### PR TITLE
chore: notice for https://github.com/aws/aws-cdk/issues/30241

### PR DESCRIPTION
**Pull request by SankyRed**
Fixes # [30093](https://github.com/aws/aws-cdk/pull/30093)

### CODE REVIEW

File: data/notices.json [hunk 1]
```diff
@@ -507,6 +507,18 @@
         }
       ],
       "schemaVersion": "1"
+    },
+    {
+      "title": "(cli): Upgrading to v2.142.0 errors out on cdk diff with `Cannot read properties of undefined`",
+      "issueNumber": 30241,
+      "overview": "The `change.resourceType` is typed as being a string, but with this change in v2.142.0 there are cases when `change.resourceType` can be undefined.",
```
Comment (colifran) [inline data/notices.json:514]: Can we change the wording here? With what change in v2.142.0? How about "A change was made to diff in CDK v2.142.0 to include changes to properties that are only present in the change set. This introduced cases where the resource type could be undefined which resulted in an error from calling 'includes' on undefined."

File: data/notices.json [hunk 2]
```diff
@@ -507,6 +507,18 @@
...
+      "components": [
+        {
+          "name": "@aws-cdk/cloudformation-diff.diff-template",
```
Comment (colifran) [inline data/notices.json:517]: This should just be cli

**Review by colifran — CHANGES_REQUESTED** [review]
We need to make sure we're giving enough context in the overview.

**Comment by SankyRed** [comment]
> We need to make sure we're giving enough context in the overview.

Updated the change as per suggestion. Thanks @colifran

**Review by colifran — APPROVED** [review]

**Pull request merged**